datafusion/core/src/execution/context.rs

```diff
@@ -1803,6 +1806,11 @@ impl SessionState {
     pub fn aggregate_functions(&self) -> &HashMap<String, Arc<AggregateUDF>> {
         &self.aggregate_functions
     }
+
+    /// Return version
+    pub fn version(&self) -> &str {
+        &self.version
```

Comment:
I think you could just call `env!("CARGO_PKG_VERSION")` here -- is there any reason to create a string to do so?

```suggestion
        env!("CARGO_PKG_VERSION")
```